docs/coding-guidelines.md

```diff
@@ -354,11 +354,13 @@ Test class names end with `Test` and live in a similar namespace as the class be
 
 #### Unit test method naming
 
-Unit test method names must be descriptive about *what is being tested*, *under what conditions*, and *what the expectations are*. Pascal casing and underscores can be used to improve readability. The following test names are correct:
+Unit test method names must be descriptive about *what is being tested*, *under what conditions*, and *what the expectations are*.
+
+Pascal casing and underscores should be used to improve readability. The following test names are correct:
```

Comment:
```suggestion
All *new* test cases should follow this convention:
Pascal casing and underscores should be used to improve readability. 

> Format: `<what is being tested>_<under what conditions>_<with what expectations>`

The following test names are correct:
```

Reply:
I'm gonna commit this, but I have an addition on top of that.